src/components/IconButton.tsx

```diff
@@ -36,6 +38,20 @@ const StyledButton = styled(Button)<{ $type: IconButtonProps["type"] }>`
             fill: var(--color-text-disabled);
           }
         `;
+      case "link":
+        return css`
+          border: 0;
+          background-color: transparent;
+          color: var(--color-button);
+          fill: var(--color-button);
+
+          &:disabled {
+            border: 0;
+            background-color: transparent;
+            color: var(--color-text-disabled);
+            fill: var(--color-text-disabled);
+          }
+        `;
```

Comment:
not sure I get this.  What's it for?

Reply:
It's part of the style definition for the `IconButton`, which can have three types now, `primary | outlined | link`. The color ramp reversal button doesn't have a background or outline, and I thought it would be easier to extend `IconButton` since it already handles making icons a nice size.

The CSS here basically just disables the borders and background color so it's just colored text, and overrides the default behavior when the button is disabled so it doesn't add a background then either.

Reply:
It doesn't, though it does handle disabled colors similarly. They differ when hovering or clicking.

Copy URL follows designs given in the Figma, while for the reversal button there wasn't an explicit design given for the hover/click behavior.

Reply:
just want to ensure we have consistent/predictable button behaviors.